### DISCUSSION TITLE
invitation from organisation

### DISCUSSION THREAD

**Discussion by ankitp-coles** [community/community#137693] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi ,
One of my organisation sent me the  invitation but by mistake I clicked on decline button.
So not able to access that repo.

now the problem is that organisation trying to add me again but he can not find my email id 

So just wanted check is there any way to delete my email id or in organisation side how to re-trigger  email again

![image](https://github.com/user-attachments/assets/534ee652-1765-4198-b139-69e03ee7205e)

**Comment by AkshayMoolya** [comment]
## Steps to Resolve GitHub Invitation Issue

If you accidentally declined an invitation to an organization and now the organization can't find your email address to re-send the invitation, follow these steps:

### 1. Check Your GitHub Email Settings

Ensure that your email address is correctly associated with your GitHub account:

1. **Log in to GitHub.**
2. Go to **Settings** (click your profile picture in the top right corner and select **Settings**).
3. In the **Emails** section, confirm that the email address the organization is trying to invite is listed.

### 2. Update Your Email Address (if needed)

If your email address is not listed, you can add it:

1. In the **Emails** section of your GitHub settings, click **Add email address**.
2. Enter the email address that should be associated with your GitHub account.
3. Follow the verification steps sent to your email.

### 3. Ask the Organization to Re-send the Invitation

The organization can try the following steps to re-send the invitation:

1. **Remove your previous invitation**:
   - An organization owner or admin can remove the pending invitation by going to the organization’s **People** settings, finding the pending invitation, and removing it.

2. **Re-send the invitation**:
   - After removing the old invitation, the organization can re-invite you by entering the correct email address again.

### 4. Contact GitHub Support

If the organization is still unable to find your email address or if there are issues with the re-invitation process, you can contact GitHub Support for assistance:

1. Go to the [GitHub Support page](https://support.github.com/).
2. Click on **Contact GitHub Support** and describe the issue.

By following these steps, you should be able to resolve the issue and gain access to the organization’s repositories.

**Comment by ankitp-coles** [comment]
Organization does not invite me because by mistake I clicked on Opt out of future invitations from organization invitation
